documentation/doxygen/Doxyfile

```diff
@@ -1,4 +1,4 @@
-# Doxyfile 1.9.0
+# Doxyfile 1.9.3
```

Comment:
This file contains many changes that are not strictly related to the correct placement of the PyROOT boxes (basically everything except for the dynamic `INPUT`, I believe). Could you place those changes in a separate commit?

Reply:
Those changes were only an upgrade to the new Doxygen version ... but fine I removed them and do only the changes related to the Python Frame.